docs/hyperlinks.md

```diff
@@ -49,6 +49,13 @@ return {
     {
       regex = [[\b[tT](\d+)\b]],
       format = "https://example.com/tasks/?t=$1",
+    },
+    
+    -- Make github like username/project paths
+    -- inside quotes and without clickable
```

Comment:
Can you add a couple of examples of the text that this will match here in the comment?

Reply:
Hopefully not too lengthy but figured I should mention that full URL's should not be picked up if they are for other services or do not reflect a user/project like path.